layouts/about/current-officers.html

```diff
@@ -0,0 +1,30 @@
+{{ define "main" }}
+
+{{ $currentOfficers := index (sort .RegularPages "Title" "desc") 0 }}
```

Comment:
Just curious, does this sorting have any effect to the order of officers shown on the page? Or is the order just determined by the order in the markdown file?

Reply:
I don't think it affects the order of the officers shown on the page. 

If I had to guess, I think it sorts all markdown files in the folder in descending order of the 'title' field and then sets 'currentOfficers' to the zeroth index in the sorted array.

Since we have only Officers.md in the folder, it doesn't really do much, but since this is just a copy of the current-execs layout, it made sense there since it displayed the most recent execs (current execs).